apps/web/components/booking/BookingListItem.tsx

```diff
@@ -123,7 +123,8 @@ function BookingListItem(booking: BookingItemProps) {
       disabled: mutation.isLoading,
     },
     // For bookings with payment, only confirm if the booking is paid for
-    ...((isPending && !booking?.eventType?.price) || (!!booking?.eventType?.price && booking.paid)
+    ...((isPending && !booking?.eventType?.metadata?.apps?.stripe?.price) ||
+    (!!booking?.eventType?.metadata?.apps?.stripe?.price && booking.paid)
```

Comment:
Sorry @Udit-takkar I should have been more specific. In the future, we can't assume all event types will use Stripe. I would look at the `getPaymentAppData` function to check if 1. A payment app is enabled, and 2. Seeing if there is a price set.